### ISSUE TITLE
error: there is no argument named `first_var`

### ISSUE THREAD

**Issue by 11hhlin1** [bbqsrc/cargo-ndk#72]
new version has this error
log::warn!("Environment variable {first_var} = '{first_path:#?}' doesn't match {var} = {path:#?

**Comment by bbqsrc** [comment]
It is warning you that your environment variables for your NDK settings conflict.

**Comment by 11hhlin1** [comment]
how to fix ，i can not build cargo-ndk

**Comment by helen-fornazier** [comment]
Hi, I have the same issue.

I'm trying to include android on the graphics mesa CI infrastructure project , but I'm getting this error https://gitlab.freedesktop.org/helen.fornazier/mesa/-/jobs/28759313#L4592

could you please point me to which settings would be conflicting?

thanks!

**Comment by helen-fornazier** [comment]
I just managed to compile on version 2.10.0

`cargo install --locked -j ${FDO_CI_CONCURRENT:-4} --root /usr/local --version 2.10.0 -- cargo-ndk`

but now I'm not sure if this is a regression or not, any ideas?

```
root@731599aabe5d:/app# cargo --version
cargo 1.49.0 (d00d64df9 2020-12-05)
root@731599aabe5d:/app# rustc --version
rustc 1.49.0 (e1884a8e3 2020-12-29)
```

maybe too old cargo version?

**Comment by bbqsrc** [comment]
Too old Rust. The bug is that it's using format strings which isn't supported in older Rust compilers. I'll patch it to use the old syntax.

**Comment by bbqsrc** [comment]
Fixed in `2.12.0`. @11hhlin1: apologies for not understanding the issue before. 🙂
